internal/splunk/hostid.go

```diff
@@ -92,3 +112,43 @@ func ResourceToHostID(res pdata.Resource) (HostID, bool) {
 
 	return HostID{}, false
 }
+
+func azureID(attrs pdata.AttributeMap, cloudAccount string) string {
+	var resourceGroupName string
+	if attr, ok := attrs.Get("azure.resourcegroup.name"); ok {
```

Comment:
Would it make sense to make `azure.resourcegroup.name` a constant that can be accessed both by the processor and this package? Same thing with `azure.vm.scaleset.name`.

Reply:
Yeah I think so. There's also a comment Bogdan made about possibly adding these to the semantic conventions.
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/2890#issuecomment-808350732
Will look into it.

Reply:
sgtm